apps/web-giddh/webpack.partial.js

```diff
@@ -22,9 +22,9 @@ module.exports = {
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
             'process.env.AppUrl': JSON.stringify('http://localhost:3000/'),
-            'process.env.ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
+            'process.env.ApiUrl': JSON.stringify('http://giddh-template.eu-west-1.elasticbeanstalk.com/'),
```

Comment:
Revert this change as it replaces ApiUrl to template URL so all the service calls that depend on this variable will use incorrect URL